app/templates/components/events/view/overview/general-info.hbs

```diff
@@ -2,13 +2,23 @@
   <div class="header">{{t 'General info'}}</div>
 </div>
 <div class="content">
-  <div class="header">
-    {{#if (eq data.event.state 'published')}}
-      {{t 'Your event is live at'}} <a href="{{data.event.url}}" rel="noopener noreferrer">{{data.event.url}}</a>
-    {{else}}
-      {{t 'Your event is not live yet. Make it live now'}}
-    {{/if}}
-  </div>
+  {{#if device.isMobile}}
+    <div class="ui small header">
+      {{#if (eq data.event.state 'published')}}
+        {{t 'Your event is live at'}} <a href="{{data.event.url}}" rel="noopener noreferrer">{{data.event.url}}</a>
+      {{else}}
+        {{t 'Your event is not live yet. Make it live now'}}
+      {{/if}}
+    </div>
+  {{else}}
+    <div class="header">
```

Comment:
We already had this discussion last time. You want to change one word in the template, you can use an inline if instead of repeating the entire code.

Reply:
@CosmicCoder96 resolved it.